routing_common/transit_types.cpp

```diff
@@ -61,5 +61,26 @@ bool Stop::IsEqualForTesting(Stop const & stop) const
   return m_id == stop.m_id && m_featureId == stop.m_featureId && m_transferId == stop.m_transferId &&
          m_lineIds == stop.m_lineIds && my::AlmostEqualAbs(m_point, stop.m_point, kPointsEqualEpsilon);
 }
+
+// Edge -------------------------------------------------------------------------------------------
+Edge::Edge(StopId startStopId, StopId finishStopId, double weight, LineId lineId, bool transfer,
+           vector<ShapeId> const & shapeIds)
+  : m_startStopId(startStopId)
+  , m_finishStopId(finishStopId)
+  , m_weight(weight)
+  , m_lineId(lineId)
+  , m_transfer(transfer)
+  , m_shapeIds(shapeIds)
+{
+}
+
+bool Edge::IsEqualForTesting(Edge const & edge) const
+{
+  double constexpr kWeightEqualEpsilon = 1e-2;
```

Comment:
Why the eps is so large?

Reply:
For decoding double to uint32 I have min = -1 and max = 10^7 (115 days). On the other hand weight edge is in seconds. So I guess 0.01 of second is not a big value.